modules/ui/TrainUI.py

```diff
@@ -8,6 +8,7 @@
 
 import customtkinter as ctk
 
+import modules.zluda.PATH
```

Comment:
I see a few problems with this line:
1. It executes code during the import, but doesn't actually use the import. This generates a warning when using static code analysis tools like PyCharm. The import might even be removed automatically
2. It's only executed here, but not from other scripts.

I just added a new file, `scripts/util/import_util.py`, that executes some default imports for all scripts. Can you add this initialization there, and also do an actual function call?

(same for the import in `scripts/train.py`)

Reply:
done.